src/us/kbase/workspace/database/provenance/PermanentID.java

```diff
@@ -42,8 +44,8 @@ public Optional<String> getDescription() {
 	/**
 	 * Gets the relationship between the ID and some other entity. For
 	 * example, when a {@link PermanentID] class is used to represent
-	 * objects in a {@link Resource}'s 'related_identifiers' field, this
-	 * field captures the relationship between the {@link Resource} and
+	 * objects in a {@link CitationMetadata}'s 'related_identifiers' field, this
```

Comment:
Change the name of the class representing the citation metadata to something more accurate